test/functional/p2p_addr_relay.py

```diff
@@ -10,7 +10,7 @@
     CAddress,
     NODE_NETWORK,
     NODE_WITNESS,
-    msg_addr,
+    msg_addrv2,
```

Comment:
Should we test for both `msg_addr` and `msg`addrv2`?

Reply:
Right, `p2p_addr_relay.py` is the only test that exercises `msg_addr` and if we flip it to test `msg_addrv2` then `msg_addr` will be untested.

I tried to combine both in one test, but it became too convoluted and so I copied the new test to `p2p_addrv2_relay.py` and restored the original `p2p_addr_relay.py`.